blockchain/genesis/genesis.go

```diff
@@ -172,7 +172,7 @@ type (
 // New constructs a genesis config. It loads the default values, and could be overwritten by values defined in the yaml
 // config files
 func New() (Genesis, error) {
-	def := defaultConfig()
+	def := Default
```

Comment:
No, we should still call defaultConfig()

Reply:
defaultConfig() does not append delegates,this cause "make run" server error,but Default appended delegates in initTestDefaultConfig()